src/mmw/js/src/modeling/gwlfe/quality/templates/table.html

```diff
@@ -14,22 +14,27 @@
     <tbody>
         {% for row in rows %}
             <tr>
-                {% for value in row %}
-                    {% if loop.index0 == 0 %}
-                        <td class="text-left">{{ value }}</td>
-                    {% else %}
-                        <td class="strong text-right">{{ value|round(2)|toLocaleString(2) }}</td>
+              {% set precision = default_precision %}
+              {% for value in row %}
+                {% if loop.index0 == 0 %}
```

Comment:
Consider using 4 spaces for indentation, as is the convention in the rest of this file and project. This would also improve the diff by highlighting only truly changed rows.